runtime/manual/references/contributing/building_from_source.md

```diff
@@ -171,3 +171,28 @@ cargo clean && cargo build -vv
 # Run:
 ./target/debug/deno run tests/testdata/run/002_hello.ts
 ```
+
+## Working with Multiple Crates
+
+If a change-set spans multiple Deno crates, you may want to build multiple
+crates together. It's suggested that you checkout all the required crates next
+to one another. For example:
+
+```shell
+- denoland/
+  - deno/
+  - deno_core/
+  - deno_ast/
+  - ...
+```
+
+Then you can use
+[Cargo's patch feature](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html)
+to override the default dependency paths:
+
+```shell
+cargo build --config 'patch.crates-io.deno_ast.path="../deno_ast"'
+```
```

Comment:
This might get cumbersome if you are working on something for several days - maybe it's worth to mention another option which is changing `Cargo.toml` by adding these lines:

```
[patch.crates-io]
deno_ast = { path = "../deno_ast" }
```

Reply:
Sounds good, although my auto complete in fish makes this so easy :).

Yes, I'm happy to add this too. Obviously they need to make sure they don't stage this change. Another thing to mention is that the version numbers must still match.

I'll update the PR.